data/card_catalogue/movement.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "title": "Lunge",
+    "upgrade": "Charge",
+    "tier": "blue",
+    "details": {
+      "rules": "Move up to 5 squares in any direction",
+      "flavour": "These are great for the glutes.",
+      "image": "assets/core/card/illustration/Lunge.png"
+    }
+  },
+  {
+    "title": "Charge",
+    "upgrade": "Teleport",
+    "tier": "green",
+    "details": {
+      "rules": {
+        "passive": "Gain advantage when running through obstacles",
+        "active": "Move up to 15 squares in any direction"
+      },
+      "flavour": "What are you running from?",
+      "image": "assets/core/card/illustration/Sprint.png"
+    }
+  },
```

Comment:
This used to be sprint, which had some wording issues (referenced combat).

I think running right through things is a fun direction to take the movement series

Reply:
Added this to the new movement-related issue

https://github.com/elliottomlinson/rpcg/issues/79#issuecomment-887651198